src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,53 @@
 import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface MovieCardProps {
+  movie: {
+    title: string;
+    description: string;
+    imgUrl: string;
+    imdbUrl: string;
+    imdbId: string;
+  };
```

Comment:
It's better to create a separate type for the movie
```suggestion
  movie: Movie;
```